diksha/profile.json

```diff
@@ -1,5 +1,6 @@
 {
     "active_profile": "diksha",
+    "db" : "dk.db",
```

Comment:
This should be under `available_profiles.diksha`

Reply:
done